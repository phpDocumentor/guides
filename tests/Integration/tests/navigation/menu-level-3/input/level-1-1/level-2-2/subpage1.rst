====================
Subpage 1, Level 2-2
====================

Lorem Ipsum Dolor.

..  menu::
    :titlesonly:

    /*
    /*/index
