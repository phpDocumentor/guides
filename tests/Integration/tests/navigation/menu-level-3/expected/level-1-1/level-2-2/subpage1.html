<!-- content start -->
    <div class="section" id="subpage-1-level-2-2">
    <h1>Subpage 1, Level 2-2</h1>

            <p>Lorem Ipsum Dolor.</p>
            <div class="menu">
    <ul class="menu-level">
    <li class="toc-item"><a href="/page1.html#page-1">Page 1</a></li>

    <li class="toc-item"><a href="/page2.html#page-2">Page 2</a></li>

    <li class="toc-item active"><a href="/level-1-1/index.html#level-1-1">Level 1-1</a>        <ul class="menu-level-1">
            <li class="toc-item"><a href="/level-1-1/subpage1.html#subpage-1-level-1-1">Subpage 1, Level 1-1</a></li>

            <li class="toc-item"><a href="/level-1-1/subpage2.html#subpage-2-level-1-1">Subpage 2, Level 1-1</a></li>

            <li class="toc-item"><a href="/level-1-1/level-2-1/index.html#level-2-1">Level 2-1</a>        <ul class="menu-level-2">
            <li class="toc-item"><a href="/level-1-1/level-2-1/subpage1.html#subpage-1-level-2-1">Subpage 1, Level 2-1</a></li>

            <li class="toc-item"><a href="/level-1-1/level-2-1/subpage2.html#subpage-2-level-2-1">Subpage 2, Level 2-1</a></li>

                    </ul></li>

            <li class="toc-item active"><a href="/level-1-1/level-2-2/index.html#level-2-2">Level 2-2</a>        <ul class="menu-level-2">
            <li class="toc-item current active"><a href="/level-1-1/level-2-2/subpage1.html#subpage-1-level-2-2">Subpage 1, Level 2-2</a></li>

            <li class="toc-item"><a href="/level-1-1/level-2-2/subpage2.html#subpage-2-level-2-2">Subpage 2, Level 2-2</a></li>

                    </ul></li>

                    </ul></li>

    <li class="toc-item"><a href="/level-1-2/index.html#level-1-2">Level 1-2</a>        <ul class="menu-level-1">
            <li class="toc-item"><a href="/level-1-2/subpage1.html#subpage-1-level-1-2">Subpage 1, Level 1-2</a></li>

            <li class="toc-item"><a href="/level-1-2/subpage2.html#subpage-2-level-1-2">Subpage 2, Level 1-2</a></li>

                    </ul></li>

    </ul>
</div>

    </div>

<!-- content end -->
